bootstrap/roles/appliance-build.bootstrap/tasks/install_zfs.yml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- name: Install zfs packages | Install dependencies
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - zfs-dkms
```

Comment:
why are we pulling in "zfs-dkms"? I don't think that's necessary (it might actually be harmful).. iirc, the zfs kernel module comes from the ubuntu kernel package, not zfs-dkms.. or am I mistaken?

Reply:
As you saw above, this was requested by @basil. My understanding is that this will effectively result in a no-op on ubuntu machines since the zfs-dkms package is already installed on all machines. Having it listed here merely makes the dependency explicit. I would be fine including it or not; I was merely attempting to address review feedback.

Reply:
I think @prakashsurya's point was that this isn't actually the correct package. I was never sure that it was, hence the question mark I used when making the request.

Reply:
Update: Actually, it looks like in some cases this can cause issues, so I'm going to remove it from the file.

Reply:
Yea, I think "zfs-dkms" is actually the wrong package.. the correct package would be `linux-modules` I think:
```
$ dpkg-query -S /lib/modules/$(uname -r)/kernel/zfs/zfs/zfs.ko
linux-modules-4.15.0-1009-aws: /lib/modules/4.15.0-1009-aws/kernel/zfs/zfs/zfs.ko
```
But IMO, I don't think we gain a whole lot by explicitly listing that dependency here.